### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by minixT**
Updated Readme file and code comments to match api changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,26 +106,28 @@ As you can see: thanks to all available RxSwift operators, in a simple way you m
...
+This observable will emit next event with new value of BluetoothManager state every time state changes.
+You could easily chain it with operation you want to perform after changing to proper state. To get current bluetooth state, use `CentralManager.state`
+property. Let's see how it looks with scanning:
 ```swift
-manager.rx_state
+manager.observeState
+	.startWith(manager.state)
 	.filter { $0 == .poweredOn }
 	.timeout(3.0, scheduler)
 	.take(1)
```
Comment (paweljaneczek) [inline README.md]: I think this `take(1)` is not needed. There won't be more than 1 event due to `ensure(.poweredOn` in `CentralManager`

Comment (paweljaneczek) [reply README.md]: hmm, question is, it is possible that `CoreBluetooth` will send `poweredOn` twice?

File: Source/Peripheral.swift [hunk 2]
```diff
@@ -73,13 +76,13 @@ public class Peripheral {
         return peripheral.state == .connected
     }
 
-    ///  Current state of `Peripheral` instance described by [CBPeripheralState](https://developer.apple.com/library/ios/documentation/CoreBluetooth/Reference/CBPeripheral_Class/#//apple_ref/c/tdef/CBPeripheralState).
+    ///  Current state of `Peripheral` instance described by [CBPeripheralState](https://developer.apple.com/documentation/corebluetooth/cbperipheralstate).
```
Comment (paweljaneczek) [inline Source/Peripheral.swift]: 👍

File: README.md (hunk 1 above)
Comment (minixT) [reply README.md]: hmm, I think CoreBluetooth sends this state each time user re-enable bluetooth in his device

Comment (paweljaneczek) [reply README.md]: yeah, but when it sends `poweredOn`, `poweredOff`, `poweredOn` then subscription is disposed due to `poweredOff` so `poweredOn` won't be called.
Problem might be when we will receive `poweredOn`, `poweredOn`, than without `take(1)` we will 2x start scanning.

**Pull request merged**